src/sobesity/domain/services/scoring.py

```diff
@@ -0,0 +1,89 @@
+import logging
+
+from sobesity.domain.entities import (
+    SkillEntity,
+    SkillFilterEnitity,
+    SkillId,
+    QuestionEntity,
+    QuestionFilterEnitity,
+    QuestionId,
+    AnswerEntity,
+    AnswerFilterEnitity,
+    AnswerId,
+    TypeEntity,
+    TypeFilterEnitity,
+    TypeId,
+)
+from sobesity.domain.serializers import NotFoundSerializer
+from sobesity.domain.utils.response import bad_request_maker
+
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+class ScoringService:
+    def __init__(self, question_service, skill_service, answer_service, type_service):
+        self.question_service = question_service
+        self.skill_service = skill_service
+        self.answer_service = answer_service
+        self.type_service = type_service
+
+    def scoring(self, dict_with_questions_and_answers):
+        total_score_right = 0
+        for question_id in dict_with_questions_and_answers:
+            question = self.question_service.get_list(
+                QuestionFilterEnitity(question_ids=[question_id])
+            )
+            if question:
+                type_of_question = self.type_service.get_list(
+                    TypeFilterEnitity(type_ids=[question[0].type_id])
+                )
+
+                if type_of_question[0].name == "multiple":
```

Comment:
Move string to constant (Enum) see example in src/sobesity/infrastructure/constants.py